src/sas/qtgui/Plotting/Slicers/BoxSlicer.py

```diff
@@ -250,14 +305,24 @@ def setParams(self, params):
         :param params: a dictionary containing name of slicer parameters and
             values the user assigned to the slicer.
         """
-        self.x = float(numpy.fabs(params["x_max"]))
-        self.y = float(numpy.fabs(params["y_max"]))
+        self.xwidth = float(numpy.fabs(params["x_width"]))
+        self.ywidth = float(numpy.fabs(params["y_width"]))
```

Comment:
Why the need take the absolute value of the serialized value(s)?

Reply:
good point. I suspect not necessary. It was probably initially necessary when the logic was for a slab always centered on 0,0 and using xmax instead of xwidth. The person who first changed that logic kept the fabs and it has been carried through sense but from the code I think xmax is always positive.

We can try to remove that and see if it breaks.